child_compassion/static/mappings/child.json

```
@@ -107,15 +109,15 @@
     "SponsoredStatus": "sponsorship_status",
     "ThingsILike": {
       "field": "hobby_ids.name",
-      "search_relational_record": true
+      "search_relational_record": true,
```

You cannot have a final `,` : it will break the JSON.
```suggestion
      "search_relational_record": true
```